docs/README.md

```diff
@@ -1,3 +1,21 @@
 # Docs
 
-Documentation for IBC apps
\ No newline at end of file
+Documentation for IBC apps
+
+## Backporting to Maintained Branches
+
+Because chains are on different versions of IBC-Go, we strive to have app compatibility across older versions of IBC-Go.
```

Comment:
I think all occurrences of `IBC-Go` could be changed to `ibc-go` since that is how the repo name is formatted.

Reply:
consider it done!